package.json

```diff
@@ -32,5 +32,8 @@
     "webpack": "^5.56.1",
     "webpack-cli": "^4.8.0",
     "webpack-dev-server": "^4.3.1"
+  },
```

Comment:
- [ ] It would be better if you add the script to start the project. This helps to start the project easily in your local environment. E.g `"start": "webpack serve --open"`. Kindly add this script. 

🛠️ 😄